src/linter/report.go

```diff
@@ -25,6 +25,15 @@ func addBuiltinCheckers(reg *CheckersRegistry) {
 			After:    `$s = strip_tags($s, '<br>')`,
 		},
 
+		{
+			Name:     "nullableType",
+			Default:  true,
+			Quickfix: true,
+			Comment:  "Report not nullable string can be null.",
+			Before:   `function f(string $str = null);`,
```

Comment:
Let's change the wording. It's not just `string`.

Reply:
Done